packages/config/config/devices/0x0086/templates/aeotec_template.json

```diff
@@ -1834,6 +1834,14 @@
 			{
 				"label": "LED follows switch state",
 				"value": 2
+			},
+			{
+				"label": "Inverted LED state based on output",
+				"value": 3
+			},
+			{
+				"label": "LED always on, blinks when state changed",
+				"value": 3
 			}
```

Comment:
First, value 3 is duplicated

Second, don't edit templates which are used in multiple devices to include a change that is only correct for some firmware versions of a single device. Instead add a new template with the new options and conditionally use it depending on the firmware version. Example [here](https://zwave-js.github.io/node-zwave-js/#/config-files/conditional-settings) at the bottom.

Reply:
thank you for pointing me in the right direction,

Both devices got new firmware with same new options, adjusting to reflect that.